clar2wasm/src/tools.rs

```diff
@@ -341,8 +341,37 @@ pub fn interpret(snippet: &str) -> Result<Option<Value>, Error> {
     interpret_at(snippet, StacksEpochId::latest(), ClarityVersion::latest())
 }
 
+struct TestConfig;
+
+impl TestConfig {
+    /// Select a Clarity version based on enabled features.
+    pub fn clarity_version() -> ClarityVersion {
```

Comment:
Should we add a check to ensure at most one of those features are set? To avoid some not obvious preference in case of multiple features

Reply:
Since those test variations are meant to run from pre-prepared CI scripts, I don't think this kind of check is mandatory here.